.vscode/launch.json

```diff
@@ -5,11 +5,30 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "type": "chrome",
+      "type": "edge",
```

Comment:
what's this change for?

Reply:
To enable this VSCode extensions, https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools.

Reply:
Can you add it to the PR description?